# ERROR: 1
(module (func (get_local 0)))
# STDERR:
1:26: local variable out of range (max 0)
